spec/support/factory_helper.rb

```diff
@@ -2,8 +2,8 @@
 
 module AnsibleTowerClient
   module FactoryHelper
-    KEYS = [:description, :extra_vars, :id, :instance_id,
-            :inventory, :name, :results, :type, :url, :related].freeze
+    KEYS = [:related, :description, :extra_vars, :id, :instance_id,
```

Comment:
`:related` has to be the first because its entries decide what attributes needs to be appended with `_id`